db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,6 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def change
+    add_column :webhooks, :source, :string, default: 'any_ci'
+    Webhook.update_all source: 'any_ci'
```

Comment:
the new column is added to all rows -> all get this default value -> no update necessary

would it make sense to remove the default after and make the column not-null so we are sure they get picked deliberately and minimize db/app coupling ?